### PR TITLE
feat(csp): hashStyles option

### PR DESCRIPTION
**Pull request by vejja**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

A user must be able to skip adding hashes to the 'style-src' policy
This is because adding even one hash cancels 'unsafe-inline'

Note: this is only for SSG

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

File: src/defaultConfig.ts [hunk 1]
```diff
@@ -81,6 +81,7 @@ export const defaultSecurityConfig = (serverlUrl: string): ModuleOptions => ({
     exclude: [/node_modules/, /\.git/]
   },
   ssg: {
-    hashScripts: true
+    hashScripts: true,
```
Comment (Baroshem) [inline src/defaultConfig.ts:84]: suggestion: Could you please write some documentation for both hashScripts and hashStyles?

I think either `utils/3.ssg.md` or `getting-started/2.configuration.md`

File: src/types/index.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import type { AllowedHTTPMethods, BasicAuth, CorsOptions, RateLimiter, RequestSi
 
 export type Ssg = {
   hashScripts?: boolean;
+  hashStyles?: boolean;
```
Comment (Baroshem) [inline src/types/index.ts:9]: suggestion: Could you also please add it to the configuration documentation page where the default config is displayed?

**Comment by Baroshem** [comment]
@vejja could you please add the recommendations I mentioned? I would love to merge this PR now so that we can plan to release it with the next RC version :)

**Comment by vejja** [comment]
Hey @Baroshem : isn't it covered in the CSP section now ?

**Comment by vejja** [comment]
Let me know if this looks ok?

**Review by Baroshem — APPROVED** [review]

**Comment by Baroshem** [comment]
Nice work, thank you! :)

**Pull request merged**